static/js/public/fullRecord.js

```diff
@@ -156,6 +182,19 @@ define('fullRecord', ['module', 'jquery', 'JP2Viewer', 'StructureView', 'dataTab
 			return "";
 		}
 
+		function showBadge(data) {
+			var markedForDeletion = false;
+			var restrictedAccess = false;
+
+			if (data.status !== undefined) {
+				var restrictions = data.status.join(',').toLowerCase();
```

Comment:
`status` is an array of known values, feels a little weird/roundabout to transform it into a string for searching

Reply:
Yeah, I was using $.array, but it didn't seem to work. I can double check it.

Reply:
Hmm, it does work with $.inArray, but I'm inclined to leave it as is, because with inArray you'd have to test for, I think three conditions for restricted status: Embargoed, Staff-Only Access and Parent Has Staff-Only Access.

Reply:
okay, we'll just have to be careful to make sure the regexes can't match across statuses in the future, since they aren't paying attention to the commas at present.